drivers/unix/os_unix.cpp

```diff
@@ -443,6 +443,20 @@ int OS_Unix::get_processor_count() const {
 	return sysconf(_SC_NPROCESSORS_CONF);
 }
 
+String OS_Unix::get_processor_name() const {
+	FileAccessRef f = FileAccess::open("/proc/cpuinfo", FileAccess::READ);
```

Comment:
> Test OS.get_processor_name() on macOS.

There's no `/proc` on macOS. `sysctl` should work.

```c++
#include <sys/sysctl.h>

...

String OS_OSX::get_processor_name() const {
	char buffer[256];
	size_t buffer_len = 256;
	if (sysctlbyname("machdep.cpu.brand_string", &buffer, &buffer_len, NULL, 0) == 0) {
		return String::utf8(buffer, buffer_len);
	}
	ERR_FAIL_V_MSG("", String("Couldn't get the CPU model name. Returning an empty string."));
}
```

Returned string example:
```
Intel(R) Xeon(R) CPU E5-1620 v2 @ 3.70GHz
```

Reply:
Android do have working `/proc/cpuinfo` (at least `cat /proc/cpuinfo` works fine on the device).